src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -261,7 +277,76 @@ object DeployPlugin extends AutoPlugin {
     stageDir
   }
 
-  /* ==========>  Environment config generation.  <========== */
+  /* ==========>  Deploy config parsing.  <========== */
+
+  /** Wrapper for config common to all hosts in a deploy.
+    * @param projectName the name of the current project
+    * @param rootDeployDir the root directory to which all replicas should be deployed on every
+    *                      remote host the project is copied to
```

Comment:
I prefer using a four-space indent for continuation lines. It uses less vertical space.